client/packages/invoices/src/InboundShipment/ListView/LinkInternalOrderModal.tsx

```diff
@@ -11,27 +11,29 @@ import {
   DialogButton,
   Typography,
 } from '@openmsupply-client/common';
+import { NameRowFragment } from 'packages/system/src';
```

Comment:
```suggestion
import { NameRowFragment } from '@openmsupply-client/system';
```

No tests calling this code (yet 😁) so technically not breaking anything, but it could in future, stick with best practice for imports